README.md

```diff
@@ -24,56 +24,107 @@ This tool can increase the performance and generalizability of machine learning
 # Requirements
 
 ---
-Tested with Python 3.7.10
+Tested with Python 3.8.18 and 3.9.18
 
 Requires:
 1. Python 
 2. pip
 
-And the following additional python package:
+And the following python packages:
 1. matplotlib
 
 2. numpy
 
-3. opencv_python_headless
+3. opencv-python-headless
 
-4. scikit_learn
+4. scikit-learn
 
 5. scipy
 
 6. umap
```

Comment:
still not corrected here, please address

Reply:
Sorry, will change that.

Reply:
Yep, and umap should be changed to umap-learn here as well.